doc/sphinx-guides/source/api/auth.rst

```diff
@@ -81,6 +81,29 @@ To test if bearer tokens are working, you can try something like the following (
   
   curl -H "Authorization: Bearer $TOKEN" http://localhost:8080/api/users/:me
 
+It may happen that when you try to authenticate a user for the first time with a bearer token, it does not have an associated user account in Dataverse. In this case, it is necessary to register the user using the following endpoint:
+
+.. code-block:: bash
+
+  curl  -H "Authorization: Bearer $TOKEN" -X POST http://localhost:8080/api/users/register --data '{"termsAccepted":true}'
+
+It is essential to send a JSON that includes the property ``termsAccepted`` set to true, which indicates that you accept the terms of service of Dataverse. Otherwise, you will not be able to create an account.
+
+In this JSON, we can also include the fields ``position`` or ``affiliation``, in the same way as when we register a user through the Dataverse UI. These fields are optional, and if not provided, they will be persisted as empty in Dataverse.
```

Comment:
They are not available, at least by default.

These are not standard OIDC claims, as these properties are specific to the Dataverse domain. They cannot be found in the PersonClaims class of the Nimbus SDK neither. 

In my opinion, these properties should not be linked to the `api-bearer-auth-json-claims` flag, as they represent Dataverse-specific properties that should be provided during sign-up (if desired, since they are optional).

Reply:
I agree to leave it as is for now. In the future, we can follow the same approach as with the other fields—send them only if the IdP doesn’t provide them. However, since these aren’t standard claims, we’ll likely need to define an expected name in Dataverse for these fields, which might vary by provider.